### PR TITLE
Add JS and CSS to display img with comment

### PR DESCRIPTION
**Pull request by lilian-hunt**

### CODE REVIEW

**Review by KathleenQ — COMMENTED** [review]
LGTM

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -115,15 +122,27 @@ xhttp.onreadystatechange = function() {
...
       const button = document.createElement('input');
       button.setAttribute('class', 'btn btn-outline-secondary');
       button.setAttribute('type', 'submit');
 
       form.appendChild(feedbackPrompt);
       form.appendChild(textArea);
+      form.appendChild(imgText);
+      form.appendChild(imgInput);
       form.appendChild(breakElement);
+      form.appendChild(breakElement2);
```
Comment (jupi3200) [inline portfolio/src/main/webapp/script.js:148]: Why do we want two consecutive break elements?

Comment (lilian-hunt) [reply portfolio/src/main/webapp/script.js:148]: It's just a style preference for the page. (You also cannot append the same break element 2x so I created two of them)

**Review by aannie-y — COMMENTED** [review]
lgtm

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -58,17 +61,22 @@ function getFeedback() {
 }
 
 /** Creates an <li> element containing text. */
```
Comment (tripd22) [inline portfolio/src/main/webapp/script.js]: Nit: now it creates a \<li> element containing _a comment_.

Comment (lilian-hunt) [reply portfolio/src/main/webapp/script.js]: Just changed

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**